action.yml

```diff
@@ -33,6 +33,14 @@ inputs:
     description: 'Boolean flag. Set to true to prevent failed tests from failing the job'
     required: false
     default: false
+  only-show-failed-tests:
```

Comment:
super minor but can we rename it to `show-failed-tests-only`? this way it feels more consistent with the other inputs

Reply:
made this change